package.json

```diff
@@ -100,11 +101,11 @@
     "gh-pages": "2.0.1",
     "husky": "2.4.0",
     "lint-staged": "8.2.0",
+    "material-table": "^1.39.2",
     "prettier": "1.18.1",
     "react-scripts": "3.0.1",
     "react-test-renderer": "16.8.6",
-    "redux-devtools-extension": "^2.13.8",
-    "material-table": "^1.39.2"
```

Comment:
Why did you remove that?

Reply:
i didnot remove them by cause,i was setting up the project on my local server , may at that time some dev dependencies got remove. i will add them again